cognite/client/_api/datapoints.py

```diff
@@ -382,6 +384,10 @@ def retrieve_dataframe(
             external_id (Union[str, List[str], Dict[str, Any], List[Dict[str, Any]]]): External id or list of external
                 ids. Can also be object specifying aggregates. See example below.
             limit (int): Maximum number of datapoints to return for each time series.
+            complete (str): post-processing of the dataframe.
+                Pass 'fill' to insert missing entries into the index, and complete data where possible (supports interpolation, stepInterpolation, count, sum, totalVariation).
+                Pass 'fill,dropna' to additionally drop rows in which any aggregate for any time series has missing values (typically rows at the start and end for interpolation aggregates).
+                This guarantees that all returned dataframes have the exact same shape and no missing values anywhere, and is only supported for aggregates using any of sum, count, totalVariance, interpolation and stepInterpolation
```

Comment:
Does "This" refer to 'fill' or 'fill,dropna', or both?

Reply:
only the latter. I was trying to make a list but don't know the syntax

Reply:
Just drop the newline then.

Reply:
added some more newlines after checking readability